setup.py

```diff
@@ -13,6 +13,7 @@
     install_requires=[
         'Scrapy>=1.6',
         'scrapinghub>=2.1.0',
+        'scrapy-spider-metadata; python_version >= "3.8"'
```

Comment:
It's possible to not add this and instead rely on the stack and/or user project settings to install it.

Reply:
It looks like a safer option not to install it by default, and rely on user project instead, because this way users are in controll of dependencies (like pydantic).

Reply:
On the other hand, we might get an outated library version, which can cause some issues. But we can get it anyways, e.g. with an older stack used.